website/src/_posts/2017-08-28-timely-processing.md

```diff
@@ -316,18 +343,43 @@ new DoFn<Event, EnrichedEvent>() {
   public void onExpiry(
       OnTimerContext context,
       @StateId("buffer") BagState<Event> bufferState) {
-    if (!bufferState.isEmpty().read()) {
+    if (!bufferState.read().isEmpty()) {
```

Comment:
This is unusual. Does Python bag state not have `isEmpty` as a future to be read? The prior code is more expected, so that emptiness can be checked via a distinct underlying RPC.

Reply:
Sorry. This was my bad.